src/org/omegat/gui/theme/DelegatingLookAndFeel.java

```diff
@@ -62,8 +62,20 @@ protected static ImageIcon getIcon(String iconName) {
     protected final LookAndFeel systemLookAndFeel;
 
     public DelegatingLookAndFeel() throws Exception {
-        systemLookAndFeel = (LookAndFeel) Class.forName(UIManager.getSystemLookAndFeelClassName())
-                .getDeclaredConstructor().newInstance();
+        // When loading GTK LaF on Java16 / Linux by Class#forName method,
+        // it cause error "java.desktop does not export com.sun.java.swing.plaf.gtk"
+        // (Bug#1052)
+        String systemLaF = UIManager.getSystemLookAndFeelClassName();
+        if (systemLaF.equals("com.sun.java.swing.plaf.gtk.GTKLookAndFeel")) {
```

Comment:
I get the same thing on macOS for `com.apple.laf.AquaLookAndFeel`.

I don't think a workaround that hard-codes platform-specific LAF classnames makes sense.

Isn't `UIManager.setLookAndFeel(systemLaF)` always going to fail as well?

What is the default value of `UIManager.getLookAndFeel()`? Is it different from the LAF represented by `UIManager.getSystemLookAndFeelClassName()`?

Reply:
Because I found this, it check name first. It can always load LaF.

https://github.com/openjdk/jdk/blob/739769c8fc4b496f08a92225a12d07414537b6c0/src/jdk.jconsole/share/classes/sun/tools/jconsole/JConsole.java#L56-L70

Reply:
I removed a condition to do it on all LaFs.

When calling `UIManager.getSystemLookAndFeel()` without  `UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());` , LaF went to fall back to Metal LaF (multi-platform LaF). So both lines are mandatory.